msm/src/verifier.rs

```diff
@@ -151,6 +151,12 @@ where
                 .values()
                 .for_each(|comm| absorb_commitment(&mut fq_sponge, comm));
 
+            // FIXME @volhovm it seems that the verifier does not
```

Comment:
It does not because fixed lookup tables should be treated as public columns, in particular when implementing vector lookup. Columns used by lookups should be considered as part of the "witness" (which includes private + public inputs), and therefore, the check would be performed automatically, if public inputs are checked.

Reply:
Hm... I guess that's quite similar to what I was trying to say. I do understand that a lot of logup columns are just "witness" (like the accumulator and so on), but the actual fixed table columns should be treated similarly to the public input. Or rather as fixed selectors tbh. "Public input" is something that changes almost every time you use the circuit, but fixed table content is mostly fixed across many public inputs. So TBD.